tables/rw-rw-g1.utb

```diff
@@ -0,0 +1,135 @@
+# liblouis: Kinyarwanda Grade 1 Braille
+#
+#-name: Kinyarwanda Grade 1 Braille
+#-index-name: Kinyarwanda Grade 1 Braille
+#-display-name: Kinyarwanda Grade 1 Braille
```

Comment:
This new table should be added to the extra/generate-display-names/display-names file. Running make in that directory will then allow to generate the correct "display-name" and "index-name" tags.

Reply:
Done.